modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverrideExportImport.testcase

```diff
@@ -42,4 +42,25 @@ definition {
 		}
 	}
 
+	@description = "This test asserts that the File Upload field is required in Import Translations page."
+	@priority = "4"
+	test CanViewFileUploadFieldIsRequired {
+		task ("Given go to the Import Translations page") {
+			Portlet.gotoPortletOptions(portletOption = "Import Translations");
+		}
+
+		task ("When click Save button") {
+			Button.clickSave();
+		}
+
+		task ("Then assert error message and File Upload field is required.") {
```

Comment:
No period in description. Update it as a full sentence please, thanks.

Reply:
Done.